lib/restful_model_collection.rb

```diff
@@ -3,7 +3,7 @@
 module Nylas
   class RestfulModelCollection
 
-    attr_accessor :filters
+    attr_accessor :filters, :_api, :model_class
```

Comment:
I found this useful for debugging purposes, but I think I should probably revert this or mark them as private explicitly?

Reply:
Curious, could you expand on how this is useful for debugging?

Reply:
Sure! I do a lot of REPL driven exploration of the code, and in a few different cases as I was exploring the API I wanted to inject a different version of `_api` and/or `model_class` to get better insights into what was going on. Having these available allowed me to replace them without reaching in and changing the instance variable.